project-templates/ecs/basic/README.md

```diff
@@ -0,0 +1,217 @@
+2. Select Github under `Version Control Provider`
+
+![img_1.png](../readme-images/publish_module_vcs_ss.png)
+
+3. Select your forked repository of the no-code module created in the last step.
+
+![img_2.png](../readme-images/select_repo_ss.png)
+
+4. Click `Publish Module`
+
+![img_3.png](../readme-images/publish_module.png)
+
+5. Review the published module details
+
+![img_4.png](../readme-images/review_module_details.png)
+
+
+
+![img_8.png](../readme-images/tfc_registry_ss.png)
+
+
+## Waypoint Template Steps 
+
+TODO(Teresa): Replace figma images with proper screenshots once UI is ready.
+
+
+1. Set up your TFC Credentials in Waypoint. You will need a TFC team access token found in Terraform Cloud and your TFC Organization Name.
+![img.png](../readme-images/waypoint_tfc_creds_setup_ss.png)
+
+
+2. Create a Waypoint Project Template by filling in the fields in the Template Creation page:
+
+Your project template will be utilized to generate the waypoint hcl for developers to start off their projects with. This template will contain
+Terraform details that will be used to spin up resources that will be used for application deployment. Please see the following example project
+template: 
+
+```shell
+// this is the simple example for now
+project = "{{ .ProjectName }}"
+
+app "{{ .ProjectName }}" {
+
+  build {
+    use "pack" {}
+
+    registry {
+      use "aws-ecr" {
+        region     = var.tfc_infra.waypoint-ecs.region
+        repository = var.tfc_infra.waypoint-ecs.ecr_repository_name
+        tag        = "latest"
+      }
+    }
+  }
+
+  deploy {
+
+    # Default workspace deploys to dev
+    use "aws-ecs" {
+      count = 1
+      memory = 512
+      cpu = 256
+   
+      cluster             = var.tfc_infra.example-ecs.ecs_cluster_name
+      log_group           = var.tfc_infra.example-ecs.log_group_name
+      region              = var.tfc_infra.example-ecs.region
+    }
+  }
+}
+
+variable "tfc_infra" {
+  default = dynamic("terraform-cloud", {
+    organization = "{{ .TfcOrgName }}"
+    workspace    = "{{ .ProjectName }}"
+  })
+  type        = any
+  sensitive   = false
+  description = "all outputs from this app's tfc workspace"
+}
+```
+
+![img_1.png](../readme-images/waypoint_create_template_ss.png)
+
+
+
+3. Return to your Projects List Page. In the top right corner, select `Create Project`. From the dropdown menu, select `Create Project with template`:
+![img_3.png](../readme-images/waypoint_projects_list_ss.png)
+
+   
+4. Select the template you would like to use for this project.
+![img_4.png](../readme-images/waypoint_select_template_ss.png)
+ 
+   
+
+5. Type `project-outstanding-panda` under `Project Name`. Then click `Create Project.
+![img_6.png](../readme-images/waypoint_create_project_ss.png)
+
+   
+6. Review the project details of your newly created project! You're Ready to Go!
+![img_7.png](../readme-images/waypoint_project_details_ss.png)
+
+
+## Waypoint Template Steps via CURL commands
+
+Pre-Requisites
```

Comment:
Did you mention how to get these Pre-Requisite values? It might be worth a sub-bullet point below mentioning how to grab the values if users don't know. It might be worth also just putting in the instructions to export them then updating the command to use the exported var

Reply:
If memory serves, you added this section before we had a UI or CLI to exercise these endpoints. To my knowledge, all of these actions are now possible via the UI or CLI, so I think we can remove this section?

It's good knowledge, and maybe we want to to save it elsewhere, but I don't think we expect anyone following the basic example to be curling.